src/framework/services/filter/FilterOption.js

```diff
@@ -6,6 +6,8 @@ export default class FilterOption {
     this.comparisonType = config.comparisonType;
     this.getValue = config.getValue;
     this.name = config.name;
+    // eslint-disable-next-line
+    this.isRemovable = config.isRemovable === false ? false : true;
```

Comment:
Better put into same line like `this.isRemovable = config.isRemovable === false ? false : true; // eslint-disable-line no-unneeded-ternary`. 
But I actually like Olgas suggestion as well.. up to you.